docs-content/installing.html.md.erb

```diff
@@ -3,76 +3,65 @@ title: Installing and Configuring AppDynamics Platform Monitoring
 
-When you successfully configure AppDynamics Nozzle on PCF, a default application named **appd-nozzle**, is created on the AppDynamics Controller. Additionally, a default tier named **appd-nozzle-tier** and a default node named **appd-nozzle-node**  are created under which corresponding metrics are recorded.
+When you successfully configure AppDynamics Nozzle on PCF, a default application nozzle, named **appd-nozzle**, is created on the AppDynamics Controller. The appd-nozzle monitors the Loggregator Firehose for specified events and metrics, and streams this data to a default nozzle tier named **appd-nozzle-tier**, and a default nozzel node named **appd-nozzle-node**. You can view these correlated logs and metrics in the Metric Browser corresponding to the Nozzle application instance or resource being monitored.
 
 ## <a id='before-you-start'></a> Before You Start
 
 1. Create a [Pivotal Network](https://network.pivotal.io) account if you do not have one.
 
 1. [Set up the AppDynamics Controller](https://docs.appdynamics.com/display/PRO45/Controller+Deployment).
 
-1. Acquire at least one Go SDK Agent License.
+1. Acquire at least one AppDynamics Go SDK Agent License. See [License Management](https://docs.appdynamics.com/display/PRO45/License+Management), and [License Entitlements and Restrictions](https://docs.appdynamics.com/display/PRO45/License+Entitlements+and+Restrictions) for more information.
 
 
-## <a id='set-up-appd-nozzle'></a> Configuring AppDynamics Nozzle
+## <a id='set-up-appd-nozzle'></a> Configuring the AppDynamics Nozzle for PCF
 
 1. Log in to the Ops Manager.
 
-1. If the AppDynamics Platform Monitoring tile does not appear in the Installation Dashboard, click **Import a Product** in the left-side menu and import the AppDynamics Platform Monitoring tile.
+1. If the AppDynamics Platform Monitoring tile does not appear in the Ops Manager Installation Dashboard, click **Import a Product** in the left-side menu to import the AppDynamics Platform Monitoring tile.
 
 1. Double-click the **AppDynamics Platform Monitoring for PCF** tile in the Installation Dashboard.
    A Settings tab is displayed.
 
 1. In the **Settings** tab, select **AppDynamics Nozzle Configuration**.
 ![Settings tab for AppDynamics Nozzle Configuration](./images/appd-nozzle-config.png)
 
-1. Specify the AppDynamics Nozzle configuration settings. All fields are mandatory. You cannot add multiple Nozzle configurations.
+1. Specify the AppDynamics Nozzle configuration settings. All fields are mandatory. You cannot add multiple Nozzle configurations. Click **Save**.
+
+* **UAA Token Endpoint**: The User Account and Authentication Service (UAA) API authentication, `token_endpoint` URL value.
+* **Doppler Logging Endpoint**: The doppler firehose endpoint address, `doppler_logging_endpoint` value.
```

Comment:
For this Environment variable it says in the User Interface of the Settings tab, select AppDynamics Nozzle Configuration it asks for a Doppler Logging Endpoint. In the cf command it is also listed as `doppler-logging-endpoint`. 
For this page, and the Overriding Configuration page it also mentioned a Doppler Traffic Controller URL, I'm unclear as to why we have these two different terms?